test/powershell/Modules/Microsoft.PowerShell.Management/Clear-Content.Tests.ps1

```diff
@@ -105,6 +105,10 @@ Describe "Clear-Content cmdlet tests" -Tags "CI" {
   }
 
   Context "Proper errors should be delivered when bad locations are specified" {
+    It "should throw when targetting a directory." {
+      { Clear-Content -Path . -ErrorAction Stop } | Should -Throw
```

Comment:
Validate the errorid:

Should -Throw -ErrorId "Your expected fully qualified error id"